cs/ccxt/base/Exchange.Number.cs

```diff
@@ -283,9 +283,9 @@ public static int PrecisionFromString(object value2)
         if (value2 == null)
             return 0;
         var value = (string)value2;
-        if (value.IndexOf('e') > -1)
+        if (value.IndexOf('e') > -1 || value.IndexOf('E') > -1)
```

Comment:
@ttodua did you test this? 
![image](https://github.com/ccxt/ccxt/assets/43336371/250c9e55-1a86-4c70-9652-b12d8ad9aca6)

Reply:
clearly this is not working 
![image](https://github.com/ccxt/ccxt/assets/43336371/0d5abd2a-c685-4782-825c-21ae11fd7306)

Reply:
ohh, in c# i had blindly replaced the regex, my bad. seems you had already pushed the fix